nflow-engine/src/main/java/com/nitorcreations/nflow/engine/workflow/instance/QueryWorkflowInstances.java

```diff
@@ -138,8 +138,8 @@ public Builder(QueryWorkflowInstances copy) {
      * @param newIds The identifiers.
      * @return this.
      */
-    public Builder addIds(Integer ... newIds) {
-      this.ids.addAll(asList(newIds));
+    public Builder addIds(List<Integer> newIds) {
```

Comment:
Other option would be to have something like this in XXXResources

       new QueryWorkflowInstances.Builder().addIds(ids.toArray(new String[ids.length]{}))